### PR TITLE
Additional approval-distribution logging

### PR DESCRIPTION
**Pull request by arkpar**

### CODE REVIEW

**Review by eskimor — APPROVED** [review]

**Comment by arkpar** [comment]
bot merge

**Comment by arkpar** [comment]
bot merge

**Comment by arkpar** [comment]
bot merge

File: node/network/approval-distribution/src/lib.rs [hunk 1]
```diff
@@ -428,11 +436,25 @@ impl State {
...
 						return;
 					}
 				}
 				hash_map::Entry::Vacant(_) => {
+					tracing::debug!(
+						target: LOG_TARGET,
+						?source,
+						?peer_id,
+						?fingerprint,
+						"Assignment from unknown peer",
```
Comment (ordian) [inline node/network/approval-distribution/src/lib.rs:456]: the peer maybe known, but we don't know if it knows about the given block (yet or it finalized some higher block)

**Review by ordian — APPROVED** [review]

File: node/network/approval-distribution/src/lib.rs [hunk 2]
```diff
@@ -428,11 +436,25 @@ impl State {
...
+						);
 						modify_reputation(ctx, peer_id, COST_DUPLICATE_MESSAGE).await;
 						return;
 					}
 				}
 				hash_map::Entry::Vacant(_) => {
+					tracing::debug!(
+						target: LOG_TARGET,
+						?source,
+						?peer_id,
```
Comment (ordian) [inline node/network/approval-distribution/src/lib.rs:454]: here and above `source` and `peer_id` is the same

**Pull request merged**